content/the-ember-times-issue-174.md

```diff
@@ -17,7 +17,7 @@ Listen and be amazed about native apps and Ember 🎙️
 ---
 
 ## [Building native apps with Ember.js 🎙️](https://audio.ember.report)
- 
+
```

Comment:
@MinThaMie I had trouble adding a suggestion to an empty line so made a commit. There was a markdownlint linter error here, it's very sensitive!